src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -160,6 +160,12 @@
             "VisibilityModifier"
     ));
 
+    // Modules that will never have xpath support ever because they not report violations
+    // till https://github.com/checkstyle/checkstyle/issues/7848
```

Comment:
please remove comment with "till".
"till" means .... when such issue become closed, we can remove code below.
But it is the same issue as you fix. Such block of code will exists till we remove SuppressWarningsHolder from repo, we do not have issue on this for now, it is far future.